core/dbt/cli/example.py

```diff
@@ -17,4 +17,4 @@
     # initialize the runner with pre-loaded profile and project, you can also pass in a preloaded manifest
     dbt = dbtRunner(profile=profile, project=project)
     # run the command, this will use the pre-loaded profile and project instead of loading
-    res, success = dbt.invoke(cli_args)
+    res = dbt.invoke(cli_args)
```

Comment:
This actually changed our previous interface a bit, two other places someone should update once this is merged
- dbt-cloud([link](https://github.com/dbt-labs/dbt-cloud/blob/05f5ead27b43ebe062e1df7fa9098e06b5e2c021/sinter/clients/dbt.py#L1618))
- dbt-server([link](https://github.com/dbt-labs/dbt-server/blob/33c602a88616ceff29d1921bf6740d576ad60a01/dbt_server/services/dbt_service.py#L298), [link](https://github.com/dbt-labs/dbt-server/blob/9e41f67dabc2aaf4db593200604f6d79c2527777/dbt_server/views.py#L292))

I probably will do the first one and CC @racheldaniel for the second one.

Reply:
Thank you!

Reply:
As discussed during quick huddle last week - I'm okay with changing the interface here. It's only being used in a few places, and we can handle those changes. Unless we think the previous interface (returning & unpacking a tuple) is strictly better.